plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHive3OnDataLake.java

```diff
@@ -205,6 +205,42 @@ public void testInsertOverwritePartitionedAndBucketedExternalTable()
         assertOverwritePartition(externalTableName);
     }
 
+    @Test
+    public void testSyncPartitionOnTableLocatedInRootPathOfBucket()
+    {
+        String tableName = "test_sync_partition_on_table_located_in_root_path_of_bucket_" + randomNameSuffix();
+        String tableLocationInRootPathOfBucket = format("s3://%s/", bucketName);
+        byte[] contents = "hello\u0001world\nbye\u0001world".getBytes(UTF_8);
+        String dummyPartitionKey = "dummy_part";
+        String dummyPartitionValue = "dummy_val";
+        String dataFilePath = format("%s=%s/data.txt", dummyPartitionKey, dummyPartitionValue);
+        hiveMinioDataLake.getMinioClient().putObject(bucketName, contents, dataFilePath);
+
+        assertUpdate(format(
+                "CREATE TABLE %s.%s (" +
+                        "  a varchar, " +
+                        "  b varchar, " +
+                        "  %s varchar) " +
+                        "WITH (" +
+                        "   external_location='%s'," +
+                        "   partitioned_by = array['%s']," +
+                        "   format='TEXTFILE')",
+                HIVE_TEST_SCHEMA,
+                tableName,
+                dummyPartitionKey,
+                tableLocationInRootPathOfBucket,
+                dummyPartitionKey));
```

Comment:
```suggestion
        hiveMinioDataLake.getMinioClient().putObject(
                bucketName, 
                "hello\u0001world\nbye\u0001world".getBytes(UTF_8), 
                format("%s=%s/data.txt", dummyPartitionKey, dummyPartitionValue));

        assertUpdate(format(
                "CREATE TABLE %s.%s (" +
                        "  a varchar, " +
                        "  b varchar, " +
                        "  %s varchar) " +
                        "WITH (" +
                        "   external_location='%s'," +
                        "   partitioned_by = array['%s']," +
                        "   format='TEXTFILE')",
                HIVE_TEST_SCHEMA,
                tableName,
                dummyPartitionKey,
                format("s3://%s/", bucketName),
                dummyPartitionKey));
```

Reply:
Reduce the number of variables to increase the readability

Reply:
Thanks! I reflect your commmit